src/server/getters/get-markets-info.ts

```diff
@@ -6,7 +6,7 @@ import { reshapeOutcomesRowToUIOutcomeInfo, reshapeMarketsRowToUIMarketInfo, get
 import { sortDirection } from "../../utils/sort-direction";
 
 export function getMarketsInfo(db: Knex, universe: Address|null|undefined, marketIDs: Array<Address>|null|undefined, sortBy: string|null|undefined, isSortDescending: boolean|null|undefined, limit: number|null|undefined, offset: number|null|undefined, callback: (err: Error|null, result?: UIMarketsInfo) => void): void {
-  let marketsQuery: Knex.QueryBuilder = getMarketsWithReportingState(db).select("blocks.timestamp as creationTime").leftJoin("blocks", "markets.creationBlockNumber", "blocks.blockNumber").orderBy("creationTime");
+  let marketsQuery: Knex.QueryBuilder = getMarketsWithReportingState(db).orderBy("creationTime");
```

Comment:
One thing I just noticed here: `getMarketsInfo` should be using the `sortBy`, `isSortDescending`, `limit`, and `offset` parameters (like the other getters do), rather than just sorting by creation time.

Reply:
This doesn't need to be fixed as part of this issue though, it is an unrelated thing!